test/base-commands/twilio-client-command.test.js

```diff
@@ -230,5 +234,69 @@ describe('base-commands', () => {
         expect(ctx.stderr).to.contain('A fake API error');
       });
     });
+
+    describe('regional and edge support', () => {
+      const envTest = (
+        args = [],
+        { envRegion = undefined, envEdge = undefined, configEdge = undefined } = {}
+      ) => {
+        return test
+          .do(ctx => {
+            ctx.userConfig = new ConfigData();
+            if (configEdge) {
+              ctx.userConfig.edge = configEdge;
+            }
+            if (envRegion) {
+              process.env.TWILIO_REGION = envRegion;
+            }
+            if (envEdge) {
+              process.env.TWILIO_EDGE = envEdge;
+            }
+            ctx.userConfig.addProfile('default-profile', constants.FAKE_ACCOUNT_SID);
+            ctx.userConfig.addProfile('region-edge-testing', constants.FAKE_ACCOUNT_SID, 'configRegion');
```

Comment:
`configRegion` is a string literal but `configEdge` is a param?

Reply:
Well, I wanted it to be tied to that profile regardless of whether that profile's active. I'll make it a param and gave it a default value of `'configRegion'`